update-submodule.ymlでエラーになる

update-submodule.yml の Update submodules to latestでエラーになる。
やりたいことは、必ずrebaseして、新しいものを正にしたい。
```
Run git config --global user.name "github-actions[bot]"
  git config --global user.name "github-actions[bot]"
  git config --global user.email "github-actions[bot]@users.noreply.github.com"
  
  # 【重要】SSL証明書の検証をオフにしてエラーを回避する
  git config --global http.sslVerify false
  
  # # サブモジュールの最新化
  # git submodule update --remote --rebase --recursive
  git submodule status | awk '{print $2}' | while read -r sub; do
    echo "=== Processing: $sub ==="
    pushd "$sub"
    git pull --rebase
    popd
  done
  shell: /usr/bin/bash -e {0}
=== Processing: _myapps-common ===
~/work/chatwork-memo/chatwork-memo/_myapps-common ~/work/chatwork-memo/chatwork-memo
Already up to date.
~/work/chatwork-memo/chatwork-memo
=== Processing: _template ===
~/work/chatwork-memo/chatwork-memo/_template ~/work/chatwork-memo/chatwork-memo
Auto-merging .github/workflows/deploy-production.yaml
CONFLICT (add/add): Merge conflict in .github/workflows/deploy-production.yaml
Auto-merging .github/workflows/run-and-deploy-test.yaml
CONFLICT (add/add): Merge conflict in .github/workflows/run-and-deploy-test.yaml
Rebasing (1/1)
error: could not apply cf7e456... update [skip ci]
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
hint: Disable this message with "git config set advice.mergeConflict false"
Could not apply cf7e456... # update [skip ci]
Error: Process completed with exit code 1.
```

--- a/.github/workflows/update-submodules.yml
+++ b/.github/workflows/update-submodules.yml
@@ -27,12 +27,11 @@ jobs:
           # 【重要】SSL証明書の検証をオフにしてエラーを回避する
           git config --global http.sslVerify false
 
-          # # サブモジュールの最新化
-          # git submodule update --remote --rebase --recursive
           git submodule status | awk '{print $2}' | while read -r sub; do
             echo "=== Processing: $sub ==="
             pushd "$sub"
-            git pull --rebase
+            git fetch origin
+            git rebase -X theirs "origin/$(git rev-parse --abbrev-ref HEAD)"
             popd
           done
 

